analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -341,6 +362,95 @@ public void Instance_InstanceMethodCallClearsField()
                 new SymbolicValue().WithConstraint(ObjectConstraint.NotNull));
         }
 
+        [TestMethod]
+        public void Invocation_StaticMethodCallClearsField()
+        {
+            var code = @"
+public class Sample: Base
+{
+    public static object _C1;
```

Comment:
We don't need the underscores. I'd prefer more descriptive names like 'FieldThis1`, `FieldThis2`, `FieldBase`, `FieldOtherClass`

Reply:
.. or `SampleField1`, to be aligned with `Sample` class and `SampleMethod`